distribution/lib/Standard/Test/0.0.0-dev/src/Main.enso

```diff
@@ -239,7 +239,7 @@ Any.should_equal that frames_to_skip=0 = case this == that of
 
          example_should_equal = Examples.add_1_to 1 . should_equal 2
 Error.should_equal : Any -> Assertion
-Error.should_equal _ = Panic.throw (Matched_On_Error this)
+Error.should_equal _ frames_to_skip=0 = here.fail_match_on_unexpected_error this 1+frames_to_skip
```

Comment:
Why is there the `frames_to_skip` argument? No one will use this method with the intention of using it (it's a failsafe for `Any.should_equal` after all), so why give it a specific argument?

Reply:
`Any.should_equal` also has this argument. It can be used to display better code locations, for example:

```
tester foo bar =
    foo.baz . should_equal bar.baz frames_to_skip=1

spec = 
    here.tester X1 X2
    here.tester X2 X3
    here.tester X3 X4
```

If the second test fails, thanks to the skipped frame I will get the location of line `here.tester X2 X3` instead of line `foo.baz . should_equal bar.baz` which would not tell me which case exactly has failed.